lib/controllers/PrometheusController.ts

```diff
@@ -279,29 +228,11 @@ class PrometheusPlugin {
    *
    * @param {Request} request
    */
-  async metrics(request) {
+  async metrics (request: KuzzleRequest) : Promise<string> {
     if (request.context.connection.protocol === 'http') {
-      request.response.setHeader('Content-Type', this.registry.contentType);
+      request.response.setHeader('Content-Type', this._registry.contentType);
```

Comment:
You should rather use the new [request.response.configure](https://docs.kuzzle.io/core/2/framework/classes/request-response/configure/) method